src/chapters/05/main.tex

```diff
@@ -253,13 +258,13 @@ \section{Solving with Python}\label{sec:solving-with-python}
 \label{fig:plot_no_vaccine}
 \end{figure}
 
-We observe that the number of infected individuals increases quickly, and in
+The number of infected individuals increases quickly, and in
 fact the rate of change increases as more individuals are infected. However
 this growth slows down as there are fewer susceptible individuals to infect. Due
 to the equal birth and death rates the overall population size remains constant;
-but we also see after some time period (around 300 time units) the levels of
-susceptible, infected, and recovered individuals becomes seemingly steady, and
-the illness becomes endemic. We can estimate once this steadiness occurs, around
+but after some time period (around 300 time units) the levels of
+susceptible, infected, and recovered individuals stabilise, and
```

Comment:
`seems to stabilise`?
We don't know if it does.

Reply:
Not in the sense of a theoretic steady state no (I agree) but in terms of what we can see? This modification was meant more as a language modification (I just felt it read better).